ambry-api/src/main/java/com/github/ambry/account/AccountService.java

```diff
@@ -109,22 +108,22 @@
   public boolean removeAccountUpdateConsumer(Consumer<Collection<Account>> accountUpdateConsumer);
 
   /**
-   * Add a new container to an existing account.
+   * Add a collection of containers to an existing account.
    * @param accountName the name of account in which new container will be added.
-   * @param newContainer the new container to add.
-   * @return a pair of account id and container id strings associated with existing account and new container.
+   * @param containers a collection of containers to add.
+   * @return a collection of containers that are successfully added.
```

Comment:
The Helix impl returns all containers (added or already exist).  Javadoc should mention that existing containers are ignored.